core/src/apps/bitcoin/sign_tx/tx_info.py

```diff
@@ -122,15 +124,13 @@ def get_tx_check_digest(self) -> bytes:
 # Used to keep track of the transaction currently being signed.
 class TxInfo(TxInfoBase):
     def __init__(self, signer: Signer, tx: SignTx) -> None:
-        super().__init__(signer)
-        self.tx = tx
```

Comment:
I think there was a good reason for `self.tx = tx` to be assigned here instead of in the parent class. It had to do with type checking. (Note that `make pystyle` also runs a static type-checker.)

Reply:
https://github.com/trezor/trezor-firmware/pull/2166/commits/14148f6d577059c8739b6c40e4b746da0b3424d1